index.html

```diff
@@ -8,14 +8,110 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="/src/styles/main.css" />
 
     <title>Проект</title>
   </head>
   <body>
-    <div class="container">
-      <h1 style="text-align: center">ПУСТОЙ ПРОЕКТ</h1>
-    </div>
+    <nav class="navbar">
+      <div class="navbar__content">
+        <div class="navbar__group navbar__group--left">
+          <a href="#" class="navbar__logo">
+            <img src="src/images/ikit-logo.png" class="navbar__logo-image" alt="Logo"
```

Comment:
я ж в видео показывал как с картинками работать
```suggestion
            <img src="/images/ikit-logo.png" class="navbar__logo-image" alt="Logo"
```